tests/testautotls.nim

```diff
@@ -160,19 +190,48 @@ suite "AutoTLS ACME Client":
 
     expect(ACMEError):
       discard await api.requestCheck(
-        "some-check-url", ACMECheckKind.ACMEOrderCheck, key, "kid"
+        parseUri("http://example.com/some-check-url"),
+        ACMECheckKind.ACMEOrderCheck,
+        key,
+        "kid",
       )
 
     expect(ACMEError):
       discard await api.requestCheck(
-        "some-check-url", ACMECheckKind.ACMEChallengeCheck, key, "kid"
+        parseUri("http://example.com/some-check-url"),
+        ACMECheckKind.ACMEChallengeCheck,
+        key,
+        "kid",
       )
 
     expect(ACMEError):
-      discard await api.requestCompleted("some-chal-url", key, "kid")
+      discard await api.requestCompleted(
+        parseUri("http://example.com/some-chal-url"), key, "kid"
+      )
 
     expect(ACMEError):
-      discard await api.requestFinalize("some-domain", "some-finalize-url", key, "kid")
+      discard await api.requestFinalize(
+        "some-domain", parseUri("http://example.com/some-finalize-url"), key, "kid"
+      )
 
     expect(ACMEError):
-      discard await api.requestGetOrder("some-order-url")
+      discard await api.requestGetOrder(parseUri("http://example.com/some-order-url"))
+
+suite "AutoTLS ACME Client":
```

Comment:
the key logic of this type is  `getCertificate` method. it would be nice to have test when everything is going on the happy path but `certificateFinalized` fails.